code/go/0chain.net/blobbercore/allocation/file_changer_update.go

```diff
@@ -90,7 +92,7 @@ func (nf *UpdateFileChanger) ProcessChange(ctx context.Context, change *Allocati
 		return nil, common.NewErrorf("process_update_file_change",
 			"setting file attributes: %v", err)
 	}
-
+	fmt.Println("Calculate Hash Need To Update Data !!!")
```

Comment:
This logs are not relevant here, please make it more detail or just remove

Reply:
indeed there are some left logs that I reused and were not removed yet.